tests/php/RestApi/StoreApi/Controllers/Cart.php

```diff
@@ -17,8 +17,6 @@
 class Cart extends TestCase {
 	/**
 	 * Setup test products data. Called before every test.
-	 *
-	 * @since 1.2.0
```

Comment:
could use our `@since $VID:$` placeholder in all new code (which should automatically get replaced with the release version on our build script).  In this particular case, would probably just need it in the class header?